lld/test/ELF/lto/time-trace.ll

```diff
@@ -0,0 +1,21 @@
+; REQUIRES: x86
+; RUN: llvm-as %s -o %t.o
+; Print to a specific file
```

Comment:
The command self explains and the comment is not that useful

Reply:
Done